lilypad/server/services/prompts.py

```diff
@@ -9,3 +11,19 @@ class PromptService(BaseService[PromptTable, PromptCreate]):
 
     table: type[PromptTable] = PromptTable
     create_model: type[PromptCreate] = PromptCreate
+
+    def find_prompt_by_params(self, prompt_create: PromptCreate) -> PromptTable | None:
```

Comment:
nit: let's rename this to `find_prompt_by_call_params` just to be extra clear in case there happen to be other params in the future